src/components/AppHeader/AppHeader.tsx

```diff
@@ -50,7 +50,7 @@ export const AppHeader = ({ version, theme, toggleTheme, changeScript }: AppHead
 
   const [script, setScript] = H.useLocalStorageState<string>(PSC.Script.RO, 'currentScript')
 
-  const { t } = useTranslation()
+  const { t }: { t: any } = useTranslation()

```

Comment:
work around for [this](https://github.com/mui-org/material-ui/issues/21218).

Reply:
what's wrong with a simple

const { t: any } = useTranslation()